### PR TITLE
more colors

### PR DESCRIPTION
**Pull request by bombcar**

### CODE REVIEW

File: gtnh-modpack.json [hunk 1]
```diff
@@ -720,6 +744,18 @@
       "side": "BOTH",
       "maven": "http://jenkins.usrv.eu:8081/nexus/content/repositories/releases/com/github/GTNewHorizons/HoloInventory/"
     },
+    {
+      "name": "HungerOverhaul",
```
Comment (mitchej123) [inline gtnh-modpack.json:748]: This is for pack building and we can't distribute this mod.  If you want to include this for dependencies/compiling adjust "side" to NONE

**Review by mitchej123 — CHANGES_REQUESTED** [review]

File: update_buildscript.sh [hunk 2]
```diff
@@ -1,5 +1,11 @@
-#!/bin/bash
+#!/usr/bin/env bash
 curl https://raw.githubusercontent.com/GTNewHorizons/ExampleMod1.7.10/main/build.gradle -o build.gradle -sS && echo "Build.gradle downloaded"
 curl https://raw.githubusercontent.com/GTNewHorizons/ExampleMod1.7.10/main/.github/workflows/release-tags.yml -o .github/workflows/release-tags.yml -sS --create-dirs && echo "Release Tags action downloaded" 
 curl https://raw.githubusercontent.com/GTNewHorizons/ExampleMod1.7.10/main/CODEOWNERS -o CODEOWNERS -sS && echo "CODEOWNERS downloaded"
-git update-index --chmod=+x gradlew && git add CODEOWNERS .github/workflows/release-tags.yml build.gradle && echo "Everything added to git"
+rm -f .github/scripts/test-no-crash-reports.sh
```
Comment (mitchej123) [inline update_buildscript.sh:5]: nit: Doesn't curl overwrite? Or it can with an option?

File: update_buildscript.sh [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+curl https://raw.githubusercontent.com/GTNewHorizons/ExampleMod1.7.10/main/.github/scripts/test-no-error-reports.sh -o .github/scripts/test-no-error-reports.sh -sS && echo "Test Errors script downloaded"
+curl https://raw.githubusercontent.com/GTNewHorizons/ExampleMod1.7.10/main/.github/workflows/build-and-test.yml -o .github/workflows/build-and-test.yml -sS && echo "Build and Test action downloaded"
+
+git add CODEOWNERS .github/scripts/test-no-error-reports.sh .github/workflows/release-tags.yml .github/workflows/build-and-test.yml build.gradle && echo "Everything added to git"
+git update-index --chmod=+x .github/scripts/test-no-error-reports.sh && chmod +x .github/scripts/test-no-error-reports.sh
+git update-index --chmod=+x gradlew && chmod +x gradlew
```
Comment (mitchej123) [inline update_buildscript.sh:11]: newline

File: repo-blacklist.json [hunk 4]
```diff
@@ -71,5 +68,8 @@
   "RedstoneFlux-API",
   "LavaMonsters",
   "GTNewHorizons-3.0",
-  "FastLeafDecay"
+  "FastLeafDecay",
+  "NotEnoughThaumcraftTabs",
+  "Railcraft-API",
+  "GTSU"
 ]
```
Comment (mitchej123) [inline repo-blacklist.json:75]: newline :sunglasses:

File: gtnh-modpack.json [hunk 5]
```diff
@@ -48,6 +48,18 @@
...
+    {
+      "name": "amunra",
+      "version": "0.4.10",
+      "tagged_at": "2022-02-13T12:53:03",
+      "repo_url": "https://github.com/GTNewHorizons/amunra",
+      "filename": "amunra-1.7.10-0.4.10.jar",
+      "download_url": "https://api.github.com/repos/GTNewHorizons/amunra/releases/assets/56693282",
+      "browser_download_url": "https://github.com/GTNewHorizons/amunra/releases/download/0.4.10/amunra-1.7.10-0.4.10.jar",
+      "license": "MIT License",
+      "side": "BOTH",
```
Comment (mitchej123) [inline gtnh-modpack.json:60]: NONE until it's included in the pack

File: update_buildscript.sh (hunk 2 above)
Comment (bombcar) [reply update_buildscript.sh:5]: the name changes from test-no-crash-reports (old buildscripts) to test-no-error-reports

File: update_buildscript.sh (hunk 3 above)
Comment (bombcar) [reply update_buildscript.sh:11]: you mean put them all on one line?

File: update_buildscript.sh (hunk 2 above)
Comment (mitchej123) [reply update_buildscript.sh:5]: ah, makes sense.

File: update_buildscript.sh (hunk 3 above)
Comment (mitchej123) [reply update_buildscript.sh:11]: no, I mean hit "return" at the end so line 12 (empty) is the end of file, not line 11

Comment (mitchej123) [reply update_buildscript.sh:11]: (No newline at end of file)

**Review by mitchej123 — APPROVED** [review]

File: update_buildscript.sh (hunk 3 above)
Comment (bombcar) [reply update_buildscript.sh]: heckles a git plug-in is striping those newlines, now to find out why

**Pull request merged**